assets/js/vue/components/search/SearchForm.vue

```diff
@@ -76,20 +76,7 @@
                 @noOfResults="changeNoOfResults"
                 :formValues="form"/>
         <section class="section-content pt-3 bg" v-else>
-            <div class="container">
-                <article class="card">
-                    <div class="card-body text-center">
-                        <h5 class="card-title">
-                            GRIIDC Datasets
```

Comment:
Removing this would remove the text on GRIIDC search as well, we need a flag variable to switch

Reply:
Made it switchable.